phases/19-capstone-projects/87-end-to-end-safety-gate/outputs/skill-end-to-end-safety-gate.md

```
@@ -0,0 +1,47 @@
+## Aggregation table
+
+| Signal state | Action |
+|---|---|
+| any high severity | block |
+| any medium severity | redact |
+| any low severity | warn |
+| nothing | allow |
+
+## Trace structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)